case_studies/dc2/train_dc2.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# run the galaxy parameters and psf parameters extration notebook
+jupyter nbconvert --execute DC2_galaxy_psf_params.ipynb --to html
+
+# run the deconvolution image generation notebook
+jupyter nbconvert --execute DC2_deconv.ipynb --to html
+
+# train base model
+bliss -cp ~/bliss/case_studies/bliss_dc2 -cn config mode=train \
+    train.trainer.logger.name=DC2_MODELS \
+    train.trainer.logger.version=base_model \
+    encoder.image_normalizer.use_deconv_channel=False \
+    encoder.image_normalizer.concat_psf_params=False \
+    encoder.do_data_augmentation = False \
+    train.pretrained_weights = None
```

Comment:
When I try running this command, I get the following error:
```
Primary config directory not found.
Check that the config directory '/home/regier/bliss/case_studies/bliss_dc2' exists and readable
```
Perhaps it should be `dc2` instead of `bliss_dc2`?